libraries/core_libs/network/src/tarcap/packets_handlers/votes_sync_packet_handler.cpp

```diff
@@ -23,8 +25,8 @@ void VotesSyncPacketHandler::validatePacketRlpFormat([[maybe_unused]] const Pack
 
 void VotesSyncPacketHandler::process(const PacketData &packet_data, const std::shared_ptr<TaraxaPeer> &peer) {
   const auto next_votes_count = packet_data.rlp_.itemCount();
-  if (next_votes_count == 0) {
-    LOG(log_er_) << "Receive 0 next votes from peer " << packet_data.from_node_id_
+  if (next_votes_count == 0 || next_votes_count > kMaxVotesInPacket) {
```

Comment:
we should probably mark the peer malicious if he sends more votes than `kMaxVotesInPacket`, but we would need to support 2 latest node versions network protocols for that ? Otherwise we would mark malicious all of the nodes, who simply did not update to the newest version yet

Reply:
yes, but we are recently introducing a lot of breaking changes. I think we need a way to handle this when we plan to do an update on mainnet and then deal with all of these breaking changes together and not one by one